core/src/test/java/feign/LoggerTest.java

```diff
@@ -157,16 +201,16 @@ public ReadTimeoutEmitsTest(Level logLevel, List<String> expectedMessages) {
       return Arrays.asList(new Object[][] {
           {Level.NONE, Collections.emptyList()},
           {Level.BASIC, Arrays.asList(
-              "\\[SendsStuff#login\\] ---> POST http://localhost:[0-9]+/ HTTP/1.1",
+              "\\[SendsStuff#login\\] ---> POST http://localhost:[0-9]+/ HTTP",
```

Comment:
same, can we make it match the original `HTTP/1.1`?

Reply:
done